src/lib/y2security/selinux.rb

```diff
@@ -122,21 +188,30 @@ def save
       return false unless configurable?
 
       Yast::Bootloader.modify_kernel_params(mode.options)
+      config_file.selinux = mode.id.to_s
+      config_file.save
 
       return true if Yast::Mode.installation
 
       Yast::Bootloader.Write
     end
 
+    # Returns needed patterns defined in the product features
+    #
+    # @return [String] defined patterns in product features to have SELinux working as expected
```

Comment:
I expect here array as it is `patterns` and also if it is separated by space it should be detail.

Reply:
Ok, will change.